release.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+if [ $# -lt 1 ]; then
+    echo "Usage:"
+    echo "./release.sh <VERSION_TO_RELEASE> [NEXT_VERSION]"
+    echo ""
+    echo "Examples:"
+    echo " When releasing a new point release:"
+    echo "   ./release.sh 0.6.3"
+    echo " When releasing a new major version:"
+    echo "   ./release.sh 0.7.0 0.8.0"
+    exit
+fi
+
+VERSION=$1
+NEXT_VERSION=$2
+
+if [ -z "$NEXT_VERSION" ]; then
+    PROMPT="About to release $VERSION. Look good? [y/n] "
+else
+    PROMPT="About to release $VERSION and bump master to $NEXT_VERSION-pre. Look good? [y/n] "
+fi
+
+read -r -p "$PROMPT" CONFIRM
+if [ "$CONFIRM" != "y" ]; then
+    exit
+fi
+
+# Make a new detached HEAD
+git checkout master
+git pull
+git checkout --detach
+make setup dist
+git add dist/
+
+# Edit package.json and bower.json to the right version
+sed -i 's|"version": "[^"]\+",|"version": "'$VERSION'",|' package.json
+sed -i 's|"version": "[^"]\+",|"version": "'$VERSION'",|' bower.json
+
+# Make the commit and tag, and push them.
+git add package.json bower.json
+git commit -m "v$VERSION"
+git tag "v$VERSION"
+git push origin "v$VERSION"
```

Comment:
Since we're not pushing this commit to anywhere except the remote tag, `git push origin` won't work.